source/endpoints/keywords.md.erb

```diff
@@ -0,0 +1,167 @@
+direction | The direction to sort by. <br><br>Possible values: asc, desc.
+search    | A general search string to filter by keywords' tags, result URL, location.
+
+## Add keywords
+
+```shell
+  curl 'https://api.nightwatch.io/api/v1/urls/URL_ID/keywords/batch_create?access_token=ACCESS_TOKEN' \\
+    -H 'Content-Type: application/json' \\
+    --data-binary '{"keywords":"keyword1\nkeyword2\nkeyword3","tags":[],"google_gl":"us","google_hl":"en","mobile":false,"desktop":true,"search_engine":"google"}' \\
+    --compressed
+```
+
+> The above command returns JSON structured like this:
+
+```json
+  [
+    { "id":1,
+      "query":"keyword1",
+      ...
+    }, {
+      "id":2,
+      "query":"keyword2",
+      ...
+    },
+    ...
+  ]
+```
+
+### HTTP Request
+
+`POST https://api.nightwatch.io/api/v1/urls/URL_ID/keywords?access_token=ACCESS_TOKEN`
+
+### Parameters
+
+Parameter | Description
+--------- | -----------
+keywords*  | Search queries separated by new lines (\n)
+google_hl* | Search language code (see <a href="https://developers.google.com/custom-search/docs/ref_languages">here</a> for available options)
+google_gl*    | Search geolocation code (see <a href="https://developers.google.com/custom-search/docs/xml_results_appendices#countryCodes">here</a> for available options)
+search_engine* | Search engine ('google', 'youtube', 'places', 'bing' or 'yahoo')
+adwords_location_id | Search location Criteria ID (see <a href="https://developers.google.com/ad-exchange/rtb/geotargeting">here</a> for available options)
+mobile | Search adjusted for mobile devices (true or false, default: true)
+
+Parameters marked with * are mandatory.
+
+## Remove keywords
+
+```shell
+curl 'https://api.nightwatch.io/api/v1/urls/URL_ID/keywords/batch_destroy?access_token=ACCESS_TOKEN' \\
+  -H 'Content-Type: application/json' \\
+  --data-binary '{"keyword_ids":["1","2"]}' \\
+  --compressed
+```
+
+> The above command returns a HTTP 204 response.
+
+
+### HTTP Request
+
+`POST https://api.nightwatch.io/api/v1/urls/URL_ID/keywords/batch_destroy?access_token=ACCESS_TOKEN`
+
+### Parameters
+
+Parameter | Description
+--------- | -----------
+keyword_ids*  | Keyword IDs for removal
+Parameters marked with * are mandatory.
+
+## Update a keyword
+
+```shell
+  curl 'https://api.nightwatch.io/api/v1/urls/URL_ID/keywords/KEYWORD_ID?access_token=ACCESS_TOKEN' \\
+    -X PUT \\
+    -H 'Content-Type: application/json; charset=UTF-8' \\
+    --data-binary '{"keyword":{"tags":["Branded"]"}}' \\
+    --compressed
+```
+
+> The above command returns a HTTP 204 response.
+
+### HTTP Request
+
+`PUT https://api.nightwatch.io/api/v1/urls/URL_ID/keywords/KEYWORD_ID?access_token=ACCESS_TOKEN`
+
+### Parameters
+
+Parameter | Description
+--------- | -----------
+keyword[tags]  | Tags
```

Comment:
Inconsistent. Parameters are always nested into a key (e.g. keyword). Why is is only mentioned in this example? Should be always or never.

Reply:
It should be more descriptive, like "An array of tags".

Reply:
Yes. Fixed.